content/docs/command-reference/remote/modify.md

```diff
@@ -250,27 +250,71 @@ For more information on configuring Azure Storage connection strings, visit
 
 Please check out
 [Setup a Google Drive DVC Remote](/doc/user-guide/setup-google-drive-remote) for
-a full guide on configuring Google Drives for use as DVC remote storage,
-including obtaining the necessary credentials, and how to form `gdrive://` URLs.
+a full guide on using Google Drive as DVC remote storage, including how to form
+`gdrive://` URLs, and high-performance/enterprise setups.
```

Comment:
same question, it's not about enterprise ... not sure about term high-perofrmance

Reply:
may be just omit ` including how to form `gdrive://` URLs, and high-performance/enterprise setups.`

Reply:
Hmmm... At least the note about GDrive URL formation is pretty important to have in both remote add and modify, I think.